czech_holidays.py

```diff
@@ -49,6 +49,11 @@ def __init__(self, year=None):
                 "Velikonoční pondělí",
                 "Easter Monday",
             ),
+             Holiday(
+                easter.year, easter.month, easter.day + 4,
```

Comment:
nemá to být `day - 3`? myslel jsem, že Velký pátek je ten před Velikonočním pondělí.